docs/guide.md

```diff
@@ -5,7 +5,9 @@ title: WebdriverIO - Developer Guide
 
 # Developer Guide
 
-Welcome to the WebdriverIO documentation. It will help you to get started fast. If you run into problems you can find help and answers on our [Gitter Channel](https://gitter.im/webdriverio/webdriverio) or you can hit me on [Twitter](https://twitter.com/webdriverio). The following will give you a short step by step introduction to get your first WebdriverIO script up and running.
+Welcome to the WebdriverIO documentation. It will help you to get started fast. If you run into problems you can find help and answers on our [Gitter Channel](https://gitter.im/webdriverio/webdriverio) or you can hit me on [Twitter](https://twitter.com/webdriverio). Also, if you encounter problems in starting up the server or running the tests after following this tutorial, ensure that the server and the geckodriver are listed in your project directory. If not, re-download them per steps 2 and 3 below.
```

Comment:
I mean to just make sure that those items are listed in the root directory of the project. A problem I had was that for some reason they weren't after I had followed the tutorial, and the errors thrown were not pointing me in the right direction. Once I made sure that both were there, I was able to get beyond the errors and start writing tests. 

The screenshot attached might help:

[webdriver-screenshot-01-07-17.pdf](https://github.com/webdriverio/webdriverio/files/691885/webdriver-screenshot-01-07-17.pdf)